met-web/src/components/engagement/form/EngagementFormTabs/Settings/EngagementInformation.tsx

```diff
@@ -19,90 +17,13 @@ const EngagementInformation = () => {
-                <MetHeader4 bold>Engagement Information</MetHeader4>
-            </Grid>
-            <Grid item xs={12} lg={6} p={1}>
-                <MetLabel>Project Name</MetLabel>
-                <TextField
-                    id="project-name"
-                    name="project_name"
-                    label=" "
-                    InputLabelProps={{
-                        shrink: false,
-                    }}
-                    value={project_metadata.project_name}
-                    variant="outlined"
-                    fullWidth
-                    onChange={handleChangeMetadata}
-                />
-            </Grid>
-            <Grid item xs={12} lg={6} p={1}>
-                <MetLabel>Client/Proponent</MetLabel>
-                <TextField
-                    id="client-name"
-                    name="client_name"
-                    value={project_metadata.client_name}
-                    variant="outlined"
-                    fullWidth
-                    onChange={handleChangeMetadata}
-                />
-            </Grid>
-            <Grid item xs={12} lg={6} p={1}>
-                <MetLabel>Project #</MetLabel>
-                <TextField
-                    id="project-id"
-                    name="project_id"
-                    value={project_id}
-                    variant="outlined"
-                    fullWidth
-                    onChange={handleChange}
-                />
-            </Grid>
-            <Grid item xs={12} lg={6} p={1}>
-                <MetLabel>Project Type</MetLabel>
-                <Select
-                    id="project-type"
-                    name="type"
-                    variant="outlined"
-                    label=" "
-                    defaultValue=""
-                    value={project_metadata.type}
-                    fullWidth
-                    size="small"
-                    onChange={handleChangeMetadata}
-                >
-                    <MenuItem value={''} sx={{ fontStyle: 'italic', height: '2em' }}>
-                        none
-                    </MenuItem>
-                    {engagementProjectTypes.map((type: string) => {
-                        return (
-                            <MenuItem key={type} value={type}>
-                                {type}
-                            </MenuItem>
-                        );
-                    })}
-                </Select>
-            </Grid>
-            <Grid item xs={6} lg={6} p={1}>
-                <MetLabel>Application #</MetLabel>
-                <TextField
-                    id="application-number"
-                    name="application_number"
-                    value={project_metadata.application_number}
-                    variant="outlined"
-                    fullWidth
-                    onChange={handleChangeMetadata}
-                />
+                <MetHeader4 bold>Engagement Metadata</MetHeader4>
```

Comment:
Are we keeping this header because we will have different metadata later?

Reply:
That was my thought, yes! And the ticket implies new metadata will go here too